pkg/probes/legacy/userdata-template.yaml

```diff
@@ -15,7 +15,7 @@ write_files:
       fi
       echo "Using IMAGE: $IMAGE" >> /var/log/userdata-output
       if [[ "${CACERT}" != "" ]]; then
-        echo "${CACERT}" | base64 --decode > /proxy.pem
+        echo "${CACERT}" > /proxy.pem
```

Comment:
@abyrne55 from your other comment - this should take care of things.

Reply:
This works as long as there's no double-quotes (or other surprise characters) in $CACERT... which is a bit big of an assumption for a user-provided input. I'd suggest reversing this change and instead encoding the $CACERT input into base64 within this probe's `GetExpandedUserData()`, or adding some input sanitization logic. Clunky, I know, but the goal of the legacy probe is to emulate the verifier's old behavior as closely as possible — which sometimes means holding our noses.

Reply:
So to be clear - we'll keep the encoding in `entry_point` but just have the new curl probe decode it back? I'm cool with that!